registration/tests/test_views.py

```diff
@@ -32,7 +32,7 @@
 from django.contrib.auth.models import User
 from django.test import TestCase
 
-from betatest.models import BetaTestApplication
+from registration.models import BetaTestApplication
```

Comment:
I noticed that the `BetaTestApplication` and `UserProfile` models doesn't inherit from `ValidateModelMixin, TimeStampedModel` - for consistency could you add them?